.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+<!--
+Thank you for contributing to jenkins-infra/charts. Before you submit this PR we'd like to
+make sure you are aware of our technical requirements and best practices:
+
+* https://github.com/jenkins-infra/charts/blob/master/CONTRIBUTING.md#technical-requirements
+* https://github.com/helm/helm-www/tree/master/content/en/docs/chart_best_practices
+
+For a quick overview across what we will look at reviewing your PR, please read
+our review guidelines:
+
+* https://github.com/jenkins-infra/charts/blob/master/REVIEW_GUIDELINES.md
+
+Following our best practices right from the start will accelerate the review process and
+help get your PR merged quicker.
+
+When updates to your PR are requested, please add new commits and do not squash the
+history. This will make it easier to identify new changes. The PR will be squashed
+anyways when it is merged. Thanks.
+
+For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.
+
+Please make sure you test your changes before you push them. Once pushed, a Jenkins job
+will run across your changes and do some initial checks and linting. These checks run
+very quickly. Please check the results. We would like these checks to pass before we
+even continue reviewing your changes.
+-->
+
+#### Is this a new chart
+- [ ] `CODEOWNERS` is updated
+- [ ] If secrets are needed, `.sops.yaml` must be updated with appropriated public gpg key ID, and full public gpg key must be provided to @olblak
+
+#### What this PR does / why we need it:
+
+#### Which issue this PR fixes
+
+  - fixes # [INFRA-XXXX](https://issues.jenkins-ci.org/projects/INFRA/issues/INFRA-XXXX)
+
+#### Special notes for your reviewer:
+
+#### Checklist
+[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
+- [ ] Chart Version bumped
```

Comment:
I still don't get the value in this, changing an image tag now means you can't do it through the UI easily as you have to change multiple files, and will the bot need to be updated to do this as well?

Reply:
if we do this possibly we should change the charts to use the latest tag and override it in the config?

as otherwise this is a PITA

Reply:
> and will the bot need to be updated to do this as well?

No it doesn't need to be updated and it's already used [here](https://github.com/jenkins-infra/charts/blob/master/updateCli/updateCli.d/jenkins-wiki-exporter.tpl#L45L15)